### ISSUE TITLE
You load pre-weights only for the first mode named unet,did you?

### ISSUE THREAD

**Issue by zwlshine** [JohnleeHIT/Brats2019#5]
In your code file "oprations.py", you extract some layers for fine-tuning, and I found these layers only from the first mode unet. Does it means just do fine-tuning for unet mode , but not do for unet_resnet mode?

New learner of tensorflow, I don't really understand fine-tuning in tensorflow, thank you very much!

**Comment by JohnleeHIT** [comment]
Yes, you are right, I only fine tune some of the layers of the mode unet,

**Comment by zwlshine** [comment]
> Yes, you are right, I only fine tune some of the layers of the mode unet,

Why not use the pre-training weights for all modes, maybe can get better dice result.
